### DISCUSSION TITLE
Window 10 start menu not working anymore after updating to 22H2 Build 22621.3007

### DISCUSSION THREAD

**Discussion by fuzzfactory0** [valinet/ExplorerPatcher#2924] [General]
None of the start menu options in explorerpatcher do anything anymore after updating windows to this version (forced update... I hate microsoft)
The start menu is back to its windows 11 appearance, and ExplorerPatcher has no control over it whatsoever. Not even the "disable the recommended section" option works anymore.

Does anyone have a solution for this?

**Comment by Amrsatrio** [comment]
Looks like you've just performed a feature upgrade. In this case, you have to reinstall EP so that its features work again on the start menu.
